docs/en/lifecycle.rst

```diff
@@ -0,0 +1,252 @@
+
+.. note::
+
+  The Wallet Attestation is short-lived. It MUST be automatically re-issued before its expiration time. For this reason, the Wallet Instance expiration 
+  transition is not considered in :numref:`fig_Wallet_Instance_States`.
+
+Transition to Installed
+^^^^^^^^^^^^^^^^^^^^^^^
+The state machine begins with the Wallet Instance installation (**WI INST**) transition, where Users download and install a Wallet Instance provided by the Wallet Provider using the 
+official app store of their device's operating system (this ensures authenticity via system checks), leading to the **Installed** state.
+
+When the state is **Installed**, the Wallet Instance MUST interact only with the Wallet Provider to be activated. When the revocation of the Wallet Instance occurs, the Wallet Instance MUST go back from **Operational** or **Valid** to **Installed**. The revocation marks the Wallet Cryptographic Hardware Key, registered during activation 
+(see “Transition to Operational” section), as not usable anymore. Revocation can occur in the following cases:
+
+  * for technical security reasons (e.g., relating to the compromise of cryptographic material); 
+  * in case of explicit User requests (e.g., due to loss, theft of the Wallet Instance);
+  * death of the User;
+  * illegal activities reported by Judicial or Supervisory Bodies.
+
+.. note::
+
+  While for the ARF the revocation of the Wallet Instance is accomplished by revoking the Wallet Attestation (see Topic 9 and Topic 38 in Annex 2), 
+  in this specification the revocation is managed differently. Being the Wallet Attestation short-lived, it does not have a status management mechanism. 
+  For this reason, the Wallet Instance revocation transition is accomplished by deleting the Wallet Cryptographic Hardware Key from the WSCD of the Wallet 
+  Instance and from the account associated with the User. This transition is completed when the Wallet Instance is online.
+
+Transition to Operational
+^^^^^^^^^^^^^^^^^^^^^^^^^
+After installation, the User opens the Wallet Instance and an activation begins (**WI ACT**). 
+At this stage, a User account MUST be created with the Wallet Provider and associated with the Wallet Instance through the Wallet Cryptographic 
+Hardware Key Tag, subject to obtaining the User's consent (see the “Wallet Instance Initialization and Registration” section for more details). 
+This association allows the User to directly request Wallet Instance revocation from the Wallet Provider, and it also allows the Wallet Provider to 
+revoke the Wallet Instance associated with that User.
+
+.. note::
+
+  As a result of the User account creation, an authentication mechanism MUST be set for the User to interact with the Wallet Provider portal. 
+  This specification mandates the use of at least a second-factor User authentication.  
```

Comment:
```suggestion
  This specification mandates the use of at least a second-factor for User authentication.  
```

we should mention an internional LoA threshold, such those defined in NIST ...

Reply:
AFAIK, this is not defined in EUDI yet. I suggest to wait for an update from EU before define more details here. What do you think?